tests/neo.UnitTests/SmartContract/Native/UT_GasToken.cs

```diff
@@ -35,6 +36,67 @@ public void TestSetup()
         [TestMethod]
         public void Check_Decimals() => NativeContract.GAS.Decimals(_snapshot).Should().Be(8);
 
+        [TestMethod]
+        public void Refuel()
+        {
+            // Prepare
+
+            var wallet = TestUtils.GenerateTestWallet();
+            var snapshot = TestBlockchain.GetTestSnapshot();
+
+            using var unlock = wallet.Unlock("");
+            var accBalance = wallet.CreateAccount();
```

Comment:
Is this deterministic?

Reply:
I know it doesn't matter... just asking :)

Reply:
No, Is not